### DISCUSSION TITLE
OTA update: Did someone acchieve OTA for STYRBAR E2001/E2002?

### DISCUSSION THREAD

**Discussion by Isy99** [Koenkk/zigbee2mqtt#12217] [Devices]
Hello folks,
neither the standard or the test server seem to have an update package.

Did someone of you acchieve an OTA? 
Everthing is running as expected, so there is no urgend need for update. But firmware is very old (1.0.024) and there is a lot of releases  documented in between.

Kind regads, Helmut

**Comment by malaakso** [comment]
There are no updates to Styrbar after 1.0.0.24. See [IKEA release notes](https://ww8.ikea.com/ikeahomesmart/releasenotes/releasenotes.html).

**Reply by Isy99** [reply]
You're rigth, understanding, that after 1.0.0.24 there were no direct (e.g. 1.0.0.25 aso) releases available upto version 1.1.15, what is mentioned in the release notes her in z2m.

**Comment by lo97** [comment]
There's a new update but it's not showing up in z2m:
Release Version: 1.23.0
19th Dec 2022
STYRBAR Remote control (V-2.4.5)
◆ Low battery indication fixed.
◆ Power reporting fixes.
◆ Fixed issue with recall scene command that is not coming through when initiating remote sync.

**Reply by malaakso** [reply]
IKEA has changed the way they version their firmwares, so the new firmware is interpreted to be older than the original. I tried to [force the update locally](https://www.zigbee2mqtt.io/guide/usage/ota_updates.html#local-ota-index-and-firmware-files), but even that didn't seem to work for some reason.

**Reply by cestnickell** [reply]
Would writing a really old version number (e.g. 0.0.0.1) to the software version attribute of the remote trigger it to update? I'm also having this issue and I am quite keen to update as I have four of these remotes and am unable to use binding with them.

**Reply by DutchCoban** [reply]
Any updates on this issue?
My Styrbar also has 1.0.0.24 and I can't get it to update.

**Reply by Luk164** [reply]
Still no solution for this?

**Reply by malaakso** [reply]
See #18515 .
